doc/version_history.rst

```diff
@@ -6,6 +6,17 @@
 Version History
 ###############
 
+v0.16.0
+=======
+
+* Use ts_tcpip OneClientReadLoopServer.
```

Comment:
That is part of the `Requires` section a few lines down. Isn't that enough?

Reply:
I prefer to explain *why* I bumped a version, when convenient. It can be helpful when reviewing history. Also I hope that the requirements section will someday go away, to be replaced by standard requirements specifications.

Reply:
Thanks. I added it before you reply.